astropy/coordinates/representation.py

```diff
@@ -348,11 +376,15 @@ def __repr__(self):
         prefixstr = '    '
         arrstr = _array2string(self._values, prefix=prefixstr)
 
+        diffstr = ''
+        if getattr(self, 'differentials', None):
+            diffstr = '\n (has differentials: {0})'.format(
+                ', '.join([repr(key) for key in self.differentials.keys()]))
```

Comment:
cc @mhvk FYI: this comment is related to why I had removed exposing the units from the `__repr__`